source/common/access_log/access_log_impl.h

```diff
@@ -208,6 +208,12 @@ class GrpcStatusFilter : public Filter {
 private:
   std::unordered_set<Grpc::Status::GrpcStatus, Grpc::GrpcStatusHash> statuses_;
   bool exclude_;
+
+  /**
+   * Converts between the config version of gRPC code and the code version of a gRPC code.
```

Comment:
Need to update this comment if you choose to rename the method.

Reply:
Addressed in 569a63a.